ecsact/cli/commands/build.cc

```diff
@@ -58,6 +58,29 @@ constexpr auto allowed_recipe_extensions = std::array{
 	".json"sv, // json works since yaml is a superset
 };
 
+auto resolve_builtin_recipe(std::string recipe_str, const char* argv[])
```

Comment:
nit: get the formatting better here

```suggestion
auto resolve_builtin_recipe( //
std::string recipe_str, const char* argv[])
```